core/filter_api.php

```diff
@@ -962,7 +962,7 @@ function filter_get_query_sort_data( array &$p_filter, $p_show_sticky, array $p_
 				$t_def = custom_field_get_definition( $t_custom_field_id );
 				$t_value_field = ( $t_def['type'] == CUSTOM_FIELD_TYPE_TEXTAREA ? 'text' : 'value' );
 				$c_cf_alias = 'custom_field_' . $t_custom_field_id;
-				$t_cf_table_alias = '{custom_field_string}_' . $t_custom_field_id;
+				$t_cf_table_alias = '{custom_field_string}_sort_' . $t_custom_field_id;
```

Comment:
Keep in mind that length of database object names are limited to 30 characters in Oracle.